GitUI/CommandsDialogs/SettingsDialog/Pages/DiffViewerSettingsPage.cs

```diff
@@ -1,14 +1,26 @@
 ﻿using GitCommands;
+using ResourceManager;
 
 namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class DiffViewerSettingsPage : SettingsPageWithHeader
     {
+        private readonly TranslationString _showDiffForAllParents = new TranslationString(@"If set, show all differences (not just the first differences).
+
+For a single selected commit, show the difference to the parent commit.
+For a single selected merge commit, show all parents.
+
+If up to four commits are multi-selected, show the difference from the first selected commits to the last selected commit.
+(If more than four commits are selected, show the difference from the first to the last selected commit only).
+
+If two commits are selected, also show the difference from the common ancestor (BASE) to the last selected.");
```

Comment:
I don't quite understand this wording.
I select commits A and B, with a common base C. The way I understand the wording I will get a diff between C and B. Why do I need A then?

Reply:
the keyword is _also_: A->B, C->B

Reply:
Ah! Thank you